renderdoc/driver/gl/gl_initstate.cpp

```diff
@@ -2426,3 +2496,15 @@ void GLResourceManager::Apply_InitialState(GLResource live, const GLInitialConte
     RDCERR("Unexpected type of resource requiring initial state");
   }
 }
+
+bool GLResourceManager::CanUseBackdoorContext(GLResource res)
```

Comment:
I think we should just use `IsResourceTrackedForPersistency` here, instead of a new function. It limits the scope of the change and for all other resources there's no real advantage to doing this - we can continue to queue prepares until the first time a 'real' sharegroup context is bound. It's only for persistent postponed resources that we might be doing the prepare at the last minute when there's no correct context bound. It also means we're not dependent on the vendor check behaviour (which can unfortunately be inconsistent).